packages-next/admin-ui/src/apollo.tsx

```diff
@@ -1,6 +1,5 @@
-/**
- * This file is exposed by the /apollo entrypoint, and helps ensure that other
- * packages import the same instance of apollo.
- */
+throw new Error(
+  'The @keystone-next/admin-ui package has been deprecated. Please import from @keystone-next/keystone/admin-ui instead.'
```

Comment:
deprecated to me means "you shouldn't use this, you still can right now but in the future, you probably won't be able to" which isn't this, could you change all these messages to say removed or merged into @keystone-next/keystone?